static/js/apps/eval_retrieval_generation/query_section.tsx

```diff
@@ -66,9 +66,15 @@ export function QuerySection(): JSX.Element {
       const answers = [];
       rowsList.forEach((rows, i) => {
         const row = rows[0];
-        const rowQ = `**${row.get(DC_QUESTION_COL)}**`;
-        const rowA = `${row.get(DC_RESPONSE_COL)} \xb7 Table ${tableIds[i]}`;
-        answers.push(`${rowQ}\n\n${rowA}`);
+        const rowQ = `<span class="dc-question">**${row.get(
+          DC_QUESTION_COL
+        )}**</span>`;
+        const rowA = `<span class="dc-stat">${row.get(
+          DC_RESPONSE_COL
+        )} \xb7 Table ${tableIds[i]}</span>`;
```

Comment:
maybe have separate vars for row.get(xx); current string formatting is a bit hard to follow.

Reply:
done